test/gtest/uct/test_stats.cc

```diff
@@ -61,34 +120,23 @@ class test_uct_stats : public uct_p2p_test {
     {
     }
 
-    void check_tx_counters(int op, int type, size_t len) {
-        uint64_t v;
-
-        v = UCS_STATS_GET_COUNTER(uct_ep(sender())->stats, op);
-        EXPECT_EQ(1UL, v);
-        v = UCS_STATS_GET_COUNTER(uct_ep(sender())->stats, type);
-        EXPECT_EQ(len, v);
-    }
-
     void check_am_rx_counters(size_t len) {
+        uint64_t iface_rx_am_init = get_cntr_init("receiver",
+                                                  UCS_PP_MAKE_STRING(UCT_IFACE_STAT_RX_AM));
+        ucs_time_t deadline = ucs::get_deadline();
```

Comment:
it's ok for c++ to define vars where you need them ( for next time)

Reply:
these are the consequences of switching from C code to C++ code :)

fixed